L1Trigger/TrackFindingTMTT/src/KFbase.cc

```diff
@@ -729,9 +729,9 @@ namespace tmtt {
     // Fixes to layermap when "maybe layer" used
     if (settings_->kfUseMaybeLayers()) {
       switch (kfEtaReg) {
-        case 5:  //case 5: B1 B2 (B3+B4)* D1 D2 D3+D4 D5+D6  -- B3 is combined with B4 and is flagged as "maybe layer"
+        case 5:  //case 5: B1 B2 (B3+B4)* D1 D2 D3 D4+D5  -- B3 is combined with B4 and is flagged as "maybe layer"
```

Comment:
Is this change a bug fix or optimisation of the KF?

Reply:
Fixing typo. There should not be D6.